3cn-ecn/nantralPlatform
# React app calendar view

# Description

Added new view for the calendar (1 day, 3 days, 1 week). The month view is not implemented yet.
Added a function to convert object key in camel case, and type into string or Date.

# Tests

1. Steps to reproduce :

- Create some events when you
- Go to the calendar in the event page.
- Click on a view button (day, 3days, week)
- Change the period of time.

2. Expected results :

The calendar adapts his display and the periods of time switch good.

# Screenshots

![image](https://user-images.githubusercontent.com/97662746/219076255-12e57a08-1c95-441f-8c15-4baf785c97d5.png)
![image](https://user-images.githubusercontent.com/97662746/219076479-e23beac3-e296-4b4f-8499-983d113fea7f.png)
![image](https://user-images.githubusercontent.com/97662746/219076559-ec4dc0d0-683a-4353-8510-6c194a3a0e5f.png)
![image](https://user-images.githubusercontent.com/97662746/219076628-cb5e0fb9-faa5-4ecd-a811-42c44afb1d3d.png)
